loader/test/support/nock-back-plugin.js

```diff
@@ -59,21 +59,49 @@ function setupNockBack(mode, fixturePath) {
  * recordings will be disallowed. Delete the recording files to record new
  * responses if the upstream server has changed.
  *
- * @param {string} name The test name
+ * @param {any} [options] Options for handling recorded data.
+ * @param {boolean} [options.ignoreQuery] Ignore querystrings in requests.
+ * @param {(any) => void} [options.prepareScope] Function that will be called
+ *        with each recorded request definition before installing them to be
+ *        replayed. Use this, for example, to make recordings match when parts
+ *        of a request (e.g. the path) are random or change from test run to
+ *        test run.
```

Comment:
The Albertsons source (and, IIRC, some others) set random querystring variables to bust caches. This, of course, makes replaying recordings hard! `ignoreQuery` is entirely my own invention here, and causes querystrings in the requests that the test makes to be ignored entirely. It’s just a special case of `prepareScope`, which itself is a *very underdocumented* Nock-back feature. It’s a callback that lets you mess with the recorded data before setting it up for replay.

Reply:
Makes sense to add this functionality, and I'm surprised it isn't already in Nock-back.

One slightly generalization that you might want to consider is allowing `ignoreQuery` to take a list of specific query parameters to remove, in case any of the others are meaningful.